cmake/CMakeBuilder.py

```diff
@@ -0,0 +1,13 @@
+from sipbuild import SetuptoolsBuilder
+
+
+class CMakeBuilder(SetuptoolsBuilder):
+    def __init__(self, project, **kwargs):
+        print("Using the CMake builder")
+        super().__init__(project, **kwargs)
+
+    def build(self):
+        """ Only Generate the source files """
```

Comment:
```suggestion
        """ Only Generate the source files """
```

Because of the tons of issues that we've had from this, I think we should have a longer writeup here as to _why_ we went for this route.

Reply:
True, Was prepping an update to to doc's and build documentation in global. But I decided to not put that in  this PR due to time constraints. 
I'm working on that documentation as we speak and I will update when ready.

Reply:
This will be updated across all three repo's: pynest2d, libsavitar and pyarcus